app/src/androidTest/java/net/skyscanner/backpack/compose/button/BpkButtonTest.kt

```diff
@@ -0,0 +1,115 @@
+package net.skyscanner.backpack.compose.button
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.interaction.PressInteraction
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.geometry.Offset
+import net.skyscanner.backpack.BpkSnapshotTest
+import net.skyscanner.backpack.BpkTestVariant
+import net.skyscanner.backpack.compose.icons.BpkIcons
+import net.skyscanner.backpack.compose.icons.lg.LongArrowRight
+import net.skyscanner.backpack.compose.icons.sm.LongArrowRight
+import org.hamcrest.Matchers.isOneOf
+import org.junit.Assume.assumeThat
+import org.junit.Assume.assumeTrue
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+@RunWith(Parameterized::class)
+class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
```

Comment:
some of the large screenshots look wonky because they go over multiple lines - can we increase the size of the screenshots?